databases/week2/meal_application.sql

```diff
@@ -0,0 +1,154 @@
+-- Part 4: Meal application
+
+-- create the database
+create database meal_share;
+
+-- create model
+use meal_share;
+
+create table Meal(
+	id INt auto_increment,
+	title varchar(30) NOT NULL,
+	description text, 
+	location varchar(50),
+	serving_time datetime,
```

Comment:
According to the data model: [https://dbdiagram.io/d/5d5bff66ced98361d6ddc18c](https://dbdiagram.io/d/5d5bff66ced98361d6ddc18c) there is no `serving_time `